desktop/config/README.md

```
@@ -0,0 +1,3 @@
+# Default Configurations
+This folder contains the default config files that will be placed in the appropriate directories when the application is installed.
+And these files can be customized by the user after installation (thru the config UI).
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)